docs/pages/concepts/logical/relationships.md

```diff
@@ -10,7 +10,7 @@ redirect_from:
 - concepts/relationships
 ---
 
-[Relationships](https://github.com/meshery/meshery/tree/master/server/meshmodel/relationships) define the nature of interaction between interconnected components in Meshery. They represent various types of connections and dependencies between components no matter the genealogy of the relationship such as parent, siblings, binding. Relationships have selectors, metadata, and optional parameters.
+[Relationships](https://docs.meshery.io/project/contributing/contributing-models) define the nature of interaction between interconnected components in Meshery. They represent various types of connections and dependencies between components no matter the genealogy of the relationship such as parent, siblings, binding. Relationships have selectors, metadata, and optional parameters.
```

Comment:
```suggestion
[Relationships]({{site.baseurl}}/project/contributing/contributing-models) define the nature of interaction between interconnected components in Meshery. They represent various types of connections and dependencies between components no matter the genealogy of the relationship such as parent, siblings, binding. Relationships have selectors, metadata, and optional parameters.
```

Reply:
@RipulHandoo use the macro for these references, please.

Also, this is the wrong place to put the contributing link.

Reply:
@leecalcote If that's the case, then there's no need to include any other links here.

Reply:
@RipulHandoo, yes, there is need for this, just not in the way that you're doing it here. How would anyone know that this goes to contributing and not to a further explanation of the concept?